docs/devenv_rhel8.md

```diff
@@ -86,6 +86,20 @@ sudo subscription-manager repos --enable rhocp-4.10-for-rhel-8-$(uname -i)-rpms
 sudo dnf install -y cri-o cri-tools
 sudo systemctl enable crio --now
 ```
+
+Download the OpenShift pull secret from the https://console.redhat.com/openshift/downloads#tool-pull-secret page and copy it into the `/etc/crio/openshift-pull-secret` file. 
+
+Run the following commands to configure CRI-O for using the pull secret when fetching container images.
+```bash
+sudo chmod 600 /etc/crio/openshift-pull-secret
+sudo mkdir -p /etc/crio/crio.conf.d/
+sudo cat > /etc/crio/crio.conf.d/openshift-pull-secret.conf << EOF
+[crio.image]
+global_auth_file="/etc/crio/openshift-pull-secret"
```

Comment:
Should we may be add this to the microshift package too?

Does crio work (start) if the openshift-pull-secret isn't in place. If it does boot may be we could keep the config there unconditionally.

Reply:
CRI-O seems to be starting OK with openshift-pull-secret file not present or empty. I will make the suggested changes